extensions/amp-a4a/0.1/test/test-amp-a4a.js

```diff
@@ -94,9 +94,15 @@ describe('amp-a4a', () => {
         sandbox.spy(AmpA4A.prototype, 'onAmpCreativeRender');
     getSigningServiceNamesMock.returns(['google']);
     xhrMockJson.withArgs(
-        'https://cdn.ampproject.org/amp-ad-verifying-keyset.json',
-        {mode: 'cors', method: 'GET'})
-    .returns(Promise.resolve({keys: [JSON.parse(validCSSAmp.publicKey)]}));
+      'https://cdn.ampproject.org/amp-ad-verifying-keyset.json',
+      {
+        mode: 'cors',
+        method: 'GET',
+        requireAmpResponseSourceOrigin: false,
+        disableAmpSourceOrigin: true,
```

Comment:
should we call it `ampCors: false`?
Overall, disabling AMP CORS is the purpose, which requires 2 things:
- Not sending the AMP_SOURCE_ORIGIN. 
- Not check response source origin.